analysis/topEFT/topeft.py

```diff
@@ -508,11 +562,11 @@ def process(self, events):
              
              # filling histos
              if var == 'invmass':
-              if   ch in ['eeeSSoffZ', 'mmmSSoffZ','eeeSSonZ', 'mmmSSonZ', '4l']: continue
+              if   ch in ['eeeSSoffZ', 'mmmSSoffZ','eeeSSonZ', 'mmmSSonZ','eeee','eeem','eemm','mmme','mmmm']: continue
```

Comment:
For readability, it might be nice to define 2lss, 3l, and 4l variables which contain these lists. That way we can do `if ch in [2lss, 4l]: continue` to make it more clear to others reading the code.